docs/admin/authorization.md

```diff
@@ -108,12 +111,19 @@ If at least one line matches the request attributes, then the request is authori
 To permit any user to do something, write a policy with the user property unset.
 To permit an action Policy with an unset namespace applies regardless of namespace.
 
+### Kubectl
+
+Kubectl uses the `/api` endpoint of api-server to negotiate client/server versions. To validate objects send to the API by create/update operations, kubectl queries certain swagger resources. For API version `v1` those would be `/swaggerapi/api/v1` & `/swaggerapi/experimental/v1`.
+
+ When users, who are restricted by namespaces, should be able to use the kubectl client, `/api` has to be exempted from authorization checks via a `"nonResourcePath":"/api"` entry in the policy file. When those users are suppossed to create or update resources, additional policies for the swagger paths `"nonResourcePath":"/swaggerapi/api/v1"`, `"nonResourcePath":"/swaggerapi/experimental/v1"` (assuming `v1` is the API version being used) have to be added (see example below).
```

Comment:
That's not really what this is doing.  This isn't exempting anything from a check.  This change is granting additional permissions to users.  I'd expect something more like "You must grant readonly access to /api for all users who need to use the kubectl client (or any other negotiating client)."